include/miral/miral/minimal_window_manager.h

```diff
@@ -72,6 +73,12 @@ class MinimalWindowManager : public WindowManagementPolicy
     /// Raises newly focused window
     void advise_focus_gained(WindowInfo const& window_info) override;
 
+    /// Change the modifer used to identify a window drag gesture
+    /// The default is mir_input_event_modifier_alt
+    /// \warning this is only expected to be called during initialization, there is no thread synchronisation
+    /// \remark Since MirAL 3.7
+    static void pointer_drag_modifier(MirInputEventModifier new_modifier);
```

Comment:
I would call this `set_pointer_drag_modifier()` to clarify it's purpose. There should be some way to use this function to disable the behavior but if there is it's not documented.

Reply:
I don't see a need for this to be a mechanism to disable the behaviour, there are many other ways to achieve that.